### PR TITLE
UI: <coll-brower> - add custom slot to add Upload button

### PR DESCRIPTION
**Pull request by iisa**
https://webarchive.jira.com/browse/WEBDEV-6535

### CODE REVIEW

File: src/sort-filter-bar/sort-filter-bar.ts [hunk 1]
```diff
@@ -130,6 +130,9 @@ export class SortFilterBar
     return html`
       <div id="container">
         <section id="sort-bar" aria-label="Sorting options">
+          <div>
+            <slot name="sort-slot-left"></slot>
+          </div>
```
Comment (latonv) [inline src/sort-filter-bar/sort-filter-bar.ts:135]: Q: Curious whether we can do away with the wrapper div here?

File: src/app-root.ts [hunk 2]
```diff
@@ -377,6 +387,12 @@ export class AppRoot extends LitElement {
           @searchTypeChanged=${this.searchTypeChanged}
           @manageModeChanged=${this.manageModeChanged}
         >
+          ${this.toggleSlots
+            ? html`<div slot="sort-slot-left">Sort Slot</div>`
+            : nothing}
+          ${this.toggleSlots
+            ? html`<div slot="facet-top-slot">Facet Slot</div>`
+            : nothing}
```
Comment (latonv) [inline src/app-root.ts:395]: Thanks for the demo additions to show these clearly!

**Review by latonv — APPROVED** [review]
LGTM 👍

File: src/app-root.ts (hunk 2 above)
Comment (nsharma123) [reply src/app-root.ts:395]: There is one more slots for list header `cb-top-slot`. lets add this one as well here.

File: src/sort-filter-bar/sort-filter-bar.ts (hunk 1 above)
Comment (nsharma123) [reply src/sort-filter-bar/sort-filter-bar.ts]: removed div wrapper

**Comment by nsharma123** [comment]
We will figure out with more convenient dev tool for toggle slot content once with done with some more slots for user-profile page.

File: src/collection-browser.ts [hunk 3]
```diff
@@ -618,6 +618,9 @@ export class CollectionBrowser
         @titleLetterChanged=${this.titleLetterSelected}
         @creatorLetterChanged=${this.creatorLetterSelected}
       >
+        <div slot="sortbar-left-slot">
+          <slot name="sortbar-left-slot"></slot>
+        </div>
```
Comment (latonv) [inline src/collection-browser.ts:623]: Q: Is the wrapper div needed here? Could we just do `<slot name="sortbar-left-slot" slot="sortbar-left-slot">` directly?

**Review by latonv — APPROVED** [review]
👍

**Pull request merged**